etc/eth-contracts/package.json

```
@@ -8,18 +8,18 @@
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.6",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
-    "hardhat-storage-layout": "^0.1.6",
     "chai": "^4.3.6",
     "eslint": "^8.18.0",
     "eslint-config-standard": "^17.0.0",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-n": "^15.0.0",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^6.0.0",
     "eth-gas-reporter": "^0.2.25",
     "ethereum-waffle": "^3.4.4",
     "ethers": "^5.6.9",
     "hardhat": "^2.9.9",
```

```suggestion
    "hardhat": "^2.10.0",
```
For node 18.4.0 devs, hardhat 2.9.9 is not compatible. I saw them adding support [here](https://github.com/NomicFoundation/hardhat/actions/runs/2322266468), and it seems they released a new version. After updating it to 2.10.0,  the error below did not show up:

```
error hardhat@2.9.9: The engine "node" is incompatible with this module. Expected version "^12.0.0 || ^14.0.0 || ^16.0.0". Got "18.4.0"
error Found incompatible module.
```

---

This should be done as a follow up PR.

---

Noted in description.